showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITLongRunningOperation.java

```diff
@@ -121,6 +124,8 @@ public void testHttpJson_LROSuccessfulResponse_doesNotExceedTotalTimeout() throw
   @Test
   public void testGRPC_LROUnsuccessfulResponse_exceedsTotalTimeout_throwsDeadlineExceededException()
       throws Exception {
+    FakeLogHandler logHandler = new FakeLogHandler();
```

Comment:
nit: Might be better to create field for FakeLogHandler that all tests can use, if possible

setUp() -> Creates it
clean() -> clears all the message so the next test can run

Reply:
Makes sense, thanks! I only added a `setUp()` since creating a new handler wipes out the old handler with its maessages